package.json

```diff
@@ -8,7 +8,7 @@
   },
   "scripts": {
     "setup": "yarn install && yarn setup:postinstall",
-    "setup:postinstall": "yarn patch-package && yarn allow-scripts",
+    "setup:postinstall": "yarn allow-scripts",
```

Comment:
Thoughts on switching to our allow-scripts yarn v3 plugin? That would remove the setup step altogether.

We've made this change elsewhere as part of the Yarn v3 migration to avoid making the contributor instructions overly complex. The old strategy of using `setup` to install deps and do post-install steps no longer works, because any attempt to run npm scripts before install fails on Yarn v3. We'd have had to say something like (use `yarn setup` to install dependencies, except the first time you'll need to run `yarn ` once first).

But when using our plugin, we can just advise that contributors use `yarn` in all cases.

Reply:
Duuude thats awesome. so yarn setup and yarn setup:postinstall are now no longer required. can you tell me what the logFilter 
```
logFilters:
  - code: YN0004
    level: discard
```
from  the module tempalte does?  why do we not care about that code?

Reply:
Also I am just going to remove the comments in the .yarnrc.yml file because using the `yarn plugin import` command erases them anyways. Seems like other cli options that touch this file might also format it so as great as it is to have them it will just be maintenance burden on future contributors.

Reply:
added the missing pieces from the module template repo's yarnrc.yml to this one, including the plugin 🎉

Reply:
The YN0004 warnings are just telling us that scripts are disabled. We handle that via allow-scripts, so those warnings aren't helpful for us.